draft-ietf-core-coap-tcp-tls.md

```diff
@@ -820,7 +820,8 @@ The following options are defined:
 
 The elective Alternative-Address Option requests the peer to instead open a connection
 of the same scheme as the present connection to the alternative transport address given.
-Its value is in the form "authority" as defined in Section 3.2 of {{RFC3986}}. 
+Its value is in the form "authority" as defined in Section 3.2 of {{RFC3986}}. Existing state

```

Comment:
Maybe we want to say "connection state" or "state related to the connection" (just to make sure people don't think application state or resource state goes away, too).

I also like to put sentences like this in parentheses: This is not really new information, just reassurance that there is nothing in the spec that would cause state transfer to happen.